src/Admin/Pool.php

```diff
@@ -343,9 +341,8 @@ public function getAdminsByGroup($group)
      *
      * @return AdminInterface|null
      *
-     * @phpstan-template T of object
-     * @phpstan-param class-string<T> $class
-     * @phpstan-return AdminInterface<T>|null
+     * @phpstan-param class-string $class
+     * @phpstan-return AdminInterface|null
```

Comment:
We can't be sure that the Admin use our class as the subject, since we could pass anything as `$adminClasses` in the constructor.